cardano-node/src/Cardano/Common/CommandLineParsers.hs

```diff
@@ -0,0 +1,364 @@
+{-# LANGUAGE RankNTypes #-}
```

Comment:
We have apparent semantic changes in this PR @Jimbo4350, and I'm afraid with this code motion it's hard to see the picture..

I'd rather see semantics and code motion in separate PRs, since their conflation makes review harder -- at least the whole picture.

Reply:
@deepfire this was due to the renaming, I'll fix it when I re-implement the parser with `attoparsec`